### DISCUSSION TITLE
s/xrlgocee9czwxt5lbdq1oa

### DISCUSSION THREAD

**Discussion by giscus** [ops-coffee/blog#6] [Announcements]
# s/xrlgocee9czwxt5lbdq1oa

这一年我们上线的自动化系统

https://blog.ops-coffee.cn/s/xrlgocee9czwxt5lbdq1oa

**Comment by Gailun90** [comment]
第三张图 IP地址暴露了。

**Reply by Gailun90** [reply]
倒数第三张

**Reply by ops-coffee** [reply]
所有的这些信息都是脱敏过的

**Comment by wdkboy** [comment]
开源了吗，我想点星星

**Reply by wdkboy** [reply]
还是你这个比较贴近业务场景

**Reply by ops-coffee** [reply]
都是实际业务使用的

**Comment by macaty** [comment]
请问Sadmin方便开源吗，我们也学习一下。

**Comment by fctee** [comment]
监控数据都是整合云上数据，那告警规则、告警方式也都是在云上配置的嘛
